Gruntfile.js

```diff
@@ -913,7 +866,7 @@ module.exports = function(grunt) {
     'exec:eslint-sw'
   ]);
 
-  grunt.registerTask('test-perf', 'Run performance-specific tests', [
+  /*grunt.registerTask('test-perf', 'Run performance-specific tests', [
```

Comment:
Instead of commenting this out, lets just remove it and raise an issue to reinstate it. Unless you're planning to come back and get this to work with 4.x?

Reply:
I think we still have some leftover code for this, but I doubt this is a relevant performance suite anymore. I think I already have an issue to "reinstate" this: https://github.com/medic/cht-core/issues/7032